scripts/o3de/o3de/manifest.py

```diff
@@ -37,6 +41,12 @@ def get_o3de_folder() -> pathlib.Path:
     return o3de_folder
 
 
+def get_o3de_user_folder() -> pathlib.Path:
```

Comment:
Disagree and commit.

Reply:
This is really more of `get_o3de_project_manager_folder`